en/21/02.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Contracts"
+actions: ['checkAnswer', 'hints']
+material: 
+  editor:
+    language: rust
+    startingCode:
+      "lib.rs": |
+        #![no_std]
+
+        multiversx_sc::imports!();
+        multiversx_sc::derive_imports!();
+
+        // start here
+    answer: |
+      #![no_std]
+
+      multiversx_sc::imports!();
+      multiversx_sc::derive_imports!();
+
+      #[multiversx_sc::contract]
+      pub trait ZombiesContract {
+        #[init]
+        fn init(&self) {}
+      }
+---
+
+
+Starting with the absolute basics:
+
+Rust is a multiparadigm  programming language focused on type safety and performance, enabling users to build fast and robust applications. A `contract` is the fundamental building block of MultiversX applications — all variables and functions belong to a contract, and this will be the starting point of all your projects.
+
+An empty contract named `HelloWorld` can be created with **mxpy** by running `mxpy contract new hello-world --template empty` from terminal and its structure would look like this:
```

Comment:
```suggestion
An empty contract named `HelloWorld` can be created with **mxpy** by running `mxpy contract new HelloWorld --template empty` from terminal and its structure would look like this:
```

Reply:
this should stay like that.

The solution directory is always snake care with `-` and the contract always gets renamed to cammel case